### ISSUE TITLE
erb-lint versions >=0.0.27 using Rainbow instead of Colorize

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/niftany#6]
```
$ bundle exec niftany
/home/travis/.rvm/gems/ruby-2.5.1/gems/niftany-0.2.0/exe/niftany:10:in `require': cannot load such file -- colorize (LoadError)
	from /home/travis/.rvm/gems/ruby-2.5.1/gems/niftany-0.2.0/exe/niftany:10:in `<top (required)>'
	from /home/travis/.rvm/gems/ruby-2.5.1/bin/niftany:23:in `load'
	from /home/travis/.rvm/gems/ruby-2.5.1/bin/niftany:23:in `<main>'
	from /home/travis/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `eval'
	from /home/travis/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `<main>'
```

Related https://github.com/Shopify/erb-lint/pull/94/commits/75361abfa137de7b4afec2dfe02040b8c9e23574

**Comment by banukutlu** [comment]
@awead this issue had been resolved with v0.3 so I am just closing this issue?

**Comment by awead** [comment]
@banukutlu sure thing. Closing now...
